### PR TITLE
Fixes #25446, #25647, #22832, #22831 - Blank entry, filtering

### PR DESCRIPTION
**Pull request by ofedoren**
Requires https://github.com/theforeman/foreman_ansible/pull/225 for `assign_roles` commands.

### CODE REVIEW

**Comment by ofedoren** [comment]
@ares, changed as you suggested.

File: lib/hammer_cli_foreman_ansible/ansible_roles.rb [hunk 1]
```diff
@@ -30,40 +48,74 @@ def execute
...
 
       build_options
     end
 
+    class PlayHostsCommand < HammerCLIForeman::Command
+      resource :hosts
+      action :multiple_play_roles
+      command_name 'play-hosts'
+
+      success_message _("Ansible roles are being played. Task's id: %{id}")
```
Comment (ares) [inline lib/hammer_cli_foreman_ansible/ansible_roles.rb]: s/Task's id/Job ID/

File: lib/hammer_cli_foreman_ansible/ansible_roles.rb [hunk 2]
```diff
@@ -30,40 +48,74 @@ def execute
...
+
+      build_options
+    end
+
+    class PlayHostgroupsCommand < HammerCLIForeman::Command
+      resource :hostgroups
+      action :multiple_play_roles
+      command_name 'play-hostgroups'
+
+      success_message _("Ansible roles are being played. Task's id: %{id}")
```
Comment (ares) [inline lib/hammer_cli_foreman_ansible/ansible_roles.rb]: s/Task's id/Job ID/

**Comment by ofedoren** [comment]
@ares, done. Thanks!

**Comment by ares** [comment]
Thanks @ofedoren, merged.

**Pull request merged**